queries.sql

```diff
@@ -8,3 +8,32 @@ select * from animal where animalname between 'Agumon' AND 'Pikachu';
 select * from animal where neutered like true;
 select * from animal where animalname not like 'Gabumon';
 select * from animal where animal_weight_kg between 10 AND 17;
+
+
+start transaction;
+update animal set species = 'unspecified';
+rollback;
+update animal set species = 'digimon' where animalname like '%mon';
+update animal set species = 'pokemon' where animalname not like '%mon';
```

Comment:
@addod19 Also can you explain more about this comment `It would be better to set the species to pokemon when the species is NULL instead we set the species to pokemon when animalname not like %mon. `

Reply:
> @addod19 Also can you explain more about this comment `It would be better to set the species to pokemon when the species is NULL instead we set the species to pokemon when animalname not like %mon. `

@Hassaanjbaig-code, yes please, I will be happy to help. Instead of `update animal set species = 'pokemon' where animalname not like '%mon';` lets change it to  ` UPDATE animals SET species = 'pokemon' WHERE species IS NULL; `

Reply:
resolved! ✔️